library/src-3.x/dotty/internal/StringContextMacro.scala

```diff
@@ -358,7 +358,7 @@ object StringContextMacro {
      */
     def checkSubtype(actualType : Type, expectedType : String, argIndex : Int, possibilities : Type*) = {
       if (possibilities.find(actualType <:< _).isEmpty)
-        reporter.argError("type mismatch;\n found : " + actualType.widen.show + "\nrequired : " + expectedType + "\n", argIndex)
+        reporter.argError("type mismatch;\n found : " + actualType.widen.dealias.show + "\nrequired : " + expectedType + "\n", argIndex)
```

Comment:
Now we got `java.lang.String`. I think that for this particular case it is good enough to just remove the prefix. 

```scala
reporter.argError("type mismatch;\n found : " + actualType.widen.show.stripPrefix("scala.Predef.") + "\nrequired : " + expectedType + "\n", argIndex)
```